src/plugin.js

```diff
@@ -61,7 +61,7 @@ const downloadButton = function(options) {
 };
 
 // Register the plugin with video.js.
-videojs.plugin('downloadButton', downloadButton);
+videojs.registerPlugin('downloadButton', downloadButton);
```

Comment:
I would prefer the cross-compatible approach https://github.com/videojs/video.js/wiki/Video.js-6-Migration-Guide#videojsplugin